examples/decoding/plot_decoding_csp_eeg.py

```diff
@@ -57,6 +57,8 @@
 
 # strip channel names
 raw.info['ch_names'] = [chn.strip('.') for chn in raw.info['ch_names']]
+for ch in raw.info['chs']:
+    ch['ch_name'] = ch['ch_name'].strip('.')
```

Comment:
how about allowing mapping in raw.rename_channels to take a callable as param?
That would avoid exposing the internals of info in an example.